Resolves #219 - missing file-destructive-push in python script

lordarcy opened the pull request:
Add file-destructive-push argument
Add --pushfiles argument to specify list of files to push
Add --removefiles argument to specify list of files to remove

I will update the wiki at https://github.com/solenopsis/Solenopsis/wiki/1.1-Usage---Python once the code has been merged.

pcon commented on scripts/lib/ant.py:
```diff
@@ -104,27 +135,20 @@ def filePush(fileList):
-        logger.critical('No files listed to push')
-        sys.exit(-1)
+    processFileList(fileList, 'sf.files2push')
+    print getFlags()
```
I don't think we want to be printing this out

pcon commented on scripts/lib/ant.py:
```diff
@@ -104,27 +135,20 @@ def filePush(fileList):
+    The list of files to push shold have already been set in the sf.files2push flag
+    """
+    processFileList(fileList, 'sf.files2remove')
+    print getFlags()
```
Same as line 139

pcon commented:
If you can please remove those two prints and re-commit I think it'll be good

lordarcy commented:
There was a print getFlags() in the original filePush(); I left it there when I refactored and added a matching one in the new fileDestructivePush(). I assumed they were there for debugging the flags that were passed to the ant build. Are you sure you want me to remove them? Those are the only two commands that have the flags printed out.

pcon commented:
Yes, please remove it. It was an oversight that the print was left in the original file-push.

pcon commented:
@lordarcy crud.  I totally missed this PR.  If you can rebase and fix the conflicts, I'll merge it.

pcon commented:
Any chance of a rebase and conflict resolution?

lordarcy commented:
@pcon I will work on this next week

pcon commented:
This was done with 374dc5bfd09cee08aa1d0de08f739ed476de1032 to fix #248
